pipeline/src/pipeline/do_pipeline.jl

```diff
@@ -12,3 +12,15 @@ function do_pipeline(pipeline::AbstractEpiAwarePipeline)
     end
     return nothing
 end
+
+"""
+Perform the pipeline for each `AbstractEpiAwarePipeline` in the given vector `pipelines`.
+
+# Arguments
+- `pipelines`: A vector of `AbstractEpiAwarePipeline` objects.
+
+"""
+function do_pipeline(pipelines::Vector{<:AbstractEpiAwarePipeline})
+    map(do_pipeline, pipelines)
```

Comment:
shouldn't this really be a dagger loop to take advantage of parallisation?

Reply:
> shouldn't this really be a dagger loop to take advantage of parallisation?

Actually I meant to make a comment on this. This still exposes parallelism, because of the `@Dagger.spawn` in the loop. But if we had more than ~120 cores I'm not sure anymore.

I guess its zero cost so I can change this. It might be a bit smarter at transition from one scenario to another.